### ISSUE TITLE
Adding multiple servers with identical hostnames

### ISSUE THREAD

**Issue by dossy** [jhuckaby/Cronicle#207]
I know the documentation states:

> All servers need to have unique hostnames (very bad things will happen otherwise).

Why is this?  I suppose this would be a feature request, but I have multiple servers each with the same hostname, different IPs, that live in different regions and are on different subnets.

In the short term, I'm going to see what I can do about renaming all of the hosts to be unique, but it would be super useful if Cronicle would generate a UUID for each server and use that as the server's unique ID, and treat hostname, IP, etc., as labels for humans, rather than using the hostname as the unique ID, which is hard (or at least inconvenient) to guarantee.

**Comment by jhuckaby** [comment]
Yeah, unfortunately Cronicle was designed with the server hostname as the primary key, so each one has to be unique.  This is a huge change, but it will definitely be addressed in the next major version.
